### DISCUSSION TITLE
Configure Download OpenAPI specification button for json vs yaml?

### DISCUSSION THREAD

**Discussion by moneil** [rohit-gohri/redocusaurus#294] [Q&A] [answered]
I may be missing something obvious here, but when using OpenAPI.json files and desiring json downloads, is there a way to configure the Download button to download the original json file vs the redocusaurus generated /redocusaurus/openapi.yaml file?

Thank you!

**Comment by rohit-gohri** [comment, marked as answer]
There is a way, you can pass `url` field along with the `spec` and it will be used as the download url. I think I accidentally deleted that option from the main docs website, but it's still there in plugin docs - https://github.com/rohit-gohri/redocusaurus/tree/8477303e143fecf79f1e8e8d77831837c22860e4/packages/docusaurus-plugin-redoc#url-optional-string-download-url

**Reply by moneil** [reply]
This worked awesome! I was able to link to my S3 bucket and open the file in a new tab from which I could save the file.

Thank you!
